pages/common/flask.md

```diff
@@ -0,0 +1,16 @@
+# flask
+
+> A general utility script for Flask applications. Loads the application defined in the $FLASK_APP environment variable.
```

Comment:
Perhaps this will read better? Then it's shell-agnostic.
```suggestion
> A general utility script for Flask applications. Loads the application defined in the `FLASK_APP` environment variable.
```

Reply:
Hmm, you're right, that's probably better.